tests/rules/parsing_method.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file Test for no-unsanitized rule
+ * @author Frederik Braun et al.
+ * @copyright 2015-2017 Mozilla Corporation. All rights reserved
+ */
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../lib/rules/parsing_method");
+const RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const eslintTester = new RuleTester();
+
+eslintTester.run("parsing_method", rule, {
+    // Examples of code that should not trigger the rule
+    // XXX this does not find z['innerHTML'] and the like.
+
+    valid: [
+        {
+            code: "var doc = DOMParser.parseFromString('static string');",
```

Comment:
That's not entirely how the function is used. People create a `DOMParser()` instance and then call `parseFromString` on it. https://developer.mozilla.org/en-US/docs/Web/API/DOMParser/parseFromString#examples

Reply:
/me feeling stupid, I even knew that, I was too focused on the other one and made the mistake